src/frameworks/contrib/ember.ts

```diff
@@ -149,22 +146,20 @@
     return value;
   } else if (Array.isArray(value)) {
     // must come before typeof value === "object" check because arrays are objects
-    return value.map(traverse);
+    return value.map((x) => traverse(x));
   } else if (typeof value === "object") {
     if (isMutableCell(value) && "value" in value) {
       return traverse(value.value);
     } else if ("_cache" in value) {
       return traverse(value._cache);
     } else if (Array.isArray(value.content)) {
       // consider arrays a traverse because knowing the property name by itself isn't useful for anything
-      return value.content.map(traverse);
-    } else {
-      return mapValues(pickExternalProps(value), recurse);
+      return value.content.map((x: any) => traverse(x));
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7734019)
```suggestion
      return value.content.map((x: unknown) => traverse(x));
```